content/en/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -46,7 +46,7 @@ Run `kubectl get nodes` to get the names of your cluster's nodes. Pick out the o
 
 If this fails with an "invalid command" error, you're likely using an older version of kubectl that doesn't have the `label` command. In that case, see the [previous version](https://github.com/kubernetes/kubernetes/blob/a053dbc313572ed60d89dae9821ecab8bfd676dc/examples/node-selection/README.md) of this guide for instructions on how to manually set labels on a node.
 
-You can verify that it worked by re-running `kubectl get nodes --show-labels` and checking that the node now has a label.
+You can verify that it worked by re-running `kubectl get nodes --show-labels` and checking that the node now has a label.You can also use `kubectl describe node "nodename"` to see the full list of labels of the given node.
```

Comment:
nit: Please add a space before the new sentence.

Reply:
Well, I will add a space.Thx!